tests/unit/orchestrator/test_orchestrator.py

```
@@ -0,0 +1,220 @@
+class TestMultiRunOrchestrator:
+    """Tests for MultiRunOrchestrator."""
+
+    @pytest.fixture
+    def mock_service_config(self):
+        """Create a mock service config."""
+        return Mock(spec=ServiceConfig)
+
+    @pytest.fixture
+    def mock_user_config(self):
+        """Create a mock user config."""
+        from aiperf.common.config import EndpointConfig, UserConfig
```

Ruff (0.14.14)

[error] Redefinition of unused `UserConfig` from line 11: `UserConfig` redefined here

Remove definition: `UserConfig`

(F811)